<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lbFont">
               <property name="text">
                <string>UI font:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combFontFamily">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Language(restart):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combLanguage"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lbAutoStartup">
               <property name="text">
                <string>Start on boot(Windows):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbAutoStartup">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Shadow effect:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbShadowEffect">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Dot size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtDotSize">
             <property name="maxLength">
              <number>30</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>UI color</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="vLayout"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Screenshot</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Image quality:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtImageQ">
               <property name="text">
                <string>100</string>
               </property>
               <property name="maxLength">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>(1-100)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QSlider" name="sliderImageQ">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Image format:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combImageType"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Save file directory:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combSaveDir">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGetSaveDir">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Save file format:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combFileFormat"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="lbDisplayFmt">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Upload</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QCheckBox" name="cbUpload">
             <property name="text">
              <string>Upload image to smms</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbProxy">
             <property name="text">
              <string>Enable proxy</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Smms Authorization: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAuth">
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Proxy Setting</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_20">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Type: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="combType">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>HTTP</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>SOCKS5</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_19">
                  <item>
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>IP: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtProxyIP">
                    <property name="text">
                     <string>127.0.0.1</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_21">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Port: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtProxyPort">
                  <property name="text">
                   <string>8888</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Hotkey</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Capture desktop:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDesktop">
               <property name="text">
                <string>Ctrl+Shift+A</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Capture desktop and copy:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDesktopCopy">
               <property name="text">
                <string>Ctrl+Shift+C</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>General capture:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFree">
               <property name="text">
                <string>F1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Setting dialog:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSettings">
               <property name="text">
                <string>Alt+Shift+S</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Quit application:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnQuit">
               <property name="text">
                <string>Alt+Shift+Q</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>About</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="hLayout"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="btnQtAbout">
             <property name="text">
              <string>About Qt</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSaveConfig">
     <property name="text">
      <string>Save configuration</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
